pkg/flare/archive.go

```
@@ -694,6 +700,22 @@ func zipDiagnose(tempDir, hostname string) error {
 	return writeScrubbedFile(f, b.Bytes())
 }
 
+func zipDatadogConnectivity(tempDir, hostname string) error {
+	var b bytes.Buffer
+
+	writer := bufio.NewWriter(&b)
+	connectivity.RunDatadogConnectivityDiagnose(writer, false) //nolint:errcheck
```

Why ignoring the error?